README.md

```diff
@@ -46,8 +46,13 @@ Here's how to get the image that Jesse built from Dockerhub and translate it to
 
 ``` bash
 cd /caldera/projects/usgs/water/iidd/datasci/lake-temp/lake-temperature-process-models
-singularity pull docker://jrossusgs/glm3r:v0.7
-# now you can see the singularity image: it is a file called glm3r_v0.7.sif
+singularity pull docker://jrossusgs/glm3r:v0.7.1
+
+# Now you can see the singularity image: it is a file called glm3r_v0.7.1.sif.
+# Create a symlink so that the launch-rstudio-container.slurm points to the new
+# container.
+rm glm3r.sif
+ln -s glm3r_v0.7.1.sif glm3r.sif
```

Comment:
The slurm launch script now uses an unversioned SIF file name. Versioning the SIF file is accomplished via a symbolic link. This allows us to play with different versions without having to edit a version-controlled file.